docker-compose-test.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.1'
+services:
+
+  auth-db:
+    image: postgres:14.3
+    networks:
+      - hmpps
+    container_name: auth-db
```

Comment:
we are still linking to auth-db (we are not in a position to pull the data into an external-users-db)